tests/tutor/assignment/test_assignment_homework.py

```diff
@@ -47,9 +47,7 @@ def test_publish_existing_unopened_homework(tutor_base_url, selenium, teacher):
     """Test publishing existing unopened homework."""
```

Comment:
the test has an action (publishing a homework) but does not have an action section (`# WHEN: `)